src/screens/ChannelListScreen/index.tsx

```diff
@@ -21,7 +20,6 @@ import {COLORS} from '../../utils/theme';
 const ChannelListScreen = ({route}) => {
   const {refresh} = useLocalDatabaseHook();
   const {checkNotificationPermission} = useRootChannelListHook();
-  const {handleFollow, isInitialFollowing} = useFollowUser();
   const {profile} = useUserAuthHook();
   const navigation = useNavigation();
   const isFocused = navigation.isFocused();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of `useFollowUser` and its related functionality seems to be in line with the PR's intent to replace it with `useFollowUserV2Hook`. However, without seeing where `useFollowUserV2Hook` is used in this file, I can't confirm if the replacement has been done correctly. Please ensure that `useFollowUserV2Hook` is being used appropriately.



<!-- This is an auto-generated comment by OSS CodeRabbit -->